iroh/src/endpoint.rs

```diff
@@ -1279,6 +1281,28 @@ impl ConnectOptions {
         self.transport_config = Some(transport_config);
         self
     }
+
+    /// Sets more [ALPN] identifiers that should be signaled as supported on connection.
```

Comment:
```suggestion
    /// Sets multiple [ALPN] identifiers to negotiate with the server.
```

Reply:
I don't like your suggestion, but it also made me dislike the current wording.
I have some ideas for something better :v:

Reply:
Pushed something that I think is an improvement.